day_10/task_1/tests/pipe_maze_tests.cpp

```diff
@@ -5,5 +5,14 @@ TEST(PipeMazeToolsShould, FindTheStartingPoint)
 {
     const PipeMaze pipe_maze = {"-L|F7", "7S-7|", "L|7||", "-L-J|", "L|-JF"};
 
-    ASSERT_EQ(find_starting_point(pipe_maze), (std::make_pair<int, int>(1, 1)));
+    ASSERT_EQ(find_starting_point(pipe_maze), MazePoint(1, 1));
+}
+
+TEST(PipeMazeToolsShould, FindeNextNeighbors)
+{
+    const PipeMaze pipe_maze = {"-L|F7", "7S-7|", "L|7||", "-L-J|", "L|-JF"};
+
+    const auto foundNeighbors = find_next_neighbors(pipe_maze);
```

Comment:
Shouldn't we be able to find the neighbors of specific pipe?
If we couldn't choose specific pipe, for which we would like to find neighbors, then I don't see the way, how we could use this function later.

```suggestion
    const MazePoint current_pipe(1,1);
    const auto foundNeighbors = find_next_neighbors(pipe_maze, current_pipe);
```